client/my-sites/customer-home/main.jsx

```diff
@@ -0,0 +1,192 @@
+import Main from 'components/main';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { getCustomizerUrl } from 'state/sites/selectors';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import DocumentHead from 'components/data/document-head';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import VerticalNav from 'components/vertical-nav';
+import VerticalNavItem from 'components/vertical-nav/item';
+
+const ActionBox = ( { action, iconSrc, label } ) => {
+	const buttonAction = 'function' === typeof action ? { onClick: action } : { href: action };
+	return (
+		<div className="customer-home__box-action">
+			<Button { ...buttonAction }>
+				<img src={ iconSrc } alt="" />
+				<span>{ label }</span>
+			</Button>
+		</div>
+	);
+};
+
+class Home extends Component {
+	static propTypes = {
+		site: PropTypes.object.isRequired,
+		siteId: PropTypes.number.isRequired,
+		siteSlug: PropTypes.string.isRequired,
+		customizeUrl: PropTypes.string.isRequired,
+	};
+
+	render() {
+		const { translate, customizeUrl, site, siteSlug } = this.props;
+
+		return (
+			<Main className="customer-home__main is-wide-layout">
+				<PageViewTracker path={ `/home/:site` } title={ translate( 'Customer Home' ) } />
+				<DocumentHead title={ translate( 'Customer Home' ) } />
+				<SidebarNavigation />
+				<div className="customer-home__layout">
+					<div className="customer-home__layout-col">
+						<Card>
+							<CardHeading>{ translate( 'My Site' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Review and update my site' ) }
+							</h6>
+							<div className="customer-home__card-button-pair">
+								<Button href={ site.URL } primary>
+									{ translate( 'View Site' ) }
+								</Button>
+								<Button href={ customizeUrl }>{ translate( 'Edit Homepage' ) }</Button>
+							</div>
+						</Card>
+						<Card className="customer-home__card-boxes">
+							<div className="customer-home__boxes">
+								<ActionBox
+									action={ () => page( `/page/${ siteSlug }` ) }
+									label={ translate( 'Add a page' ) }
+									iconSrc="/calypso/images/customer-home/page.svg"
+								/>
+								<ActionBox
+									action={ () => page( `/post/${ siteSlug }` ) }
+									label={ translate( 'Write blog post' ) }
+									iconSrc="/calypso/images/customer-home/post.svg"
+								/>
+								<ActionBox
+									action={ customizeUrl }
+									label={ translate( 'Customize theme' ) }
+									iconSrc="/calypso/images/customer-home/customize.svg"
+								/>
+								<ActionBox
+									action={ () => page( `/themes/${ siteSlug }` ) }
+									label={ translate( 'Change theme' ) }
+									iconSrc="/calypso/images/customer-home/theme.svg"
+								/>
+								<ActionBox
+									action={ customizeUrl }
+									label={ translate( 'Edit menus' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **127** times:
`translate( 'Edit Menus' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/customer-home-section)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).